source/androidx.work/work-runtime/Additions/WorkAdditions.cs

```diff
@@ -61,17 +61,17 @@ public new Builder SetInitialState(WorkInfo.State state)
             public new Builder SetInputData(Data data)
                 => base.SetInputData(data).JavaCast<Builder>();
 
-            public new Builder SetPeriodStartTime(long periodStartTime, TimeUnit timeUnit)
-                => base.SetPeriodStartTime(periodStartTime, timeUnit).JavaCast<Builder>();
+            // public new Builder SetPeriodStartTime(long periodStartTime, TimeUnit timeUnit)
```

Comment:
Why are the methods in this file commented out?  This has been exposed as public API that people may be using.

Reply:
Seems like `SetPeriodStartTime` was removed.

I checked decompiled jar and did not find anything.

```
/Users/Shared/Projects/d/X/AX/wsu-20230222/source/androidx.work/work-runtime/Additions/WorkAdditions.cs(149,25): error CS0117: 'WorkRequest.Builder' does not contain a definition for 'SetPeriodStartTime' [/Users/Shared/Projects/d/X/AX/wsu-20230222/generated/androidx.work.work-runtime/androidx.work.work-runtime.csproj]
/Users/Shared/Projects/d/X/AX/wsu-20230222/source/androidx.work/work-runtime/Additions/WorkAdditions.cs(152,25): error CS0117: 'WorkRequest.Builder' does not contain a definition for 'SetPeriodStartTime' [/Users/Shared/Projects/d/X/AX/wsu-20230222/generated/androidx.work.work-runtime/androidx.work.work-runtime.csproj]
/Users/Shared/Projects/d/X/AX/wsu-20230222/source/androidx.work/work-runtime/Additions/WorkAdditions.cs(155,25): error CS0117: 'WorkRequest.Builder' does not contain a definition for 'SetPeriodStartTime' [/Users/Shared/Projects/d/X/AX/wsu-20230222/generated/androidx.work.work-runtime/androidx.work.work-runtime.csproj]
/Users/Shared/Projects/d/X/AX/wsu-20230222/source/androidx.work/work-runtime/Additions/WorkAdditions.cs(158,25): error CS0117: 'WorkRequest.Builder' does not contain a definition for 'SetPeriodStartTime' [/Users/Shared/Projects/d/X/AX/wsu-20230222/generated/androidx.work.work-runtime/androidx.work.work-runtime.csproj]
/Users/Shared/Projects/d/X/AX/wsu-20230222/source/androidx.work/work-runtime/Additions/WorkAdditions.cs(149,25): error CS0117: 'WorkRequest.Builder' does not contain a definition for 'SetPeriodStartTime' [/Users/Shared/Projects/d/X/AX/wsu-20230222/generated/androidx.work.work-runtime/androidx.work.work-runtime.csproj]
/Users/Shared/Projects/d/X/AX/wsu-20230222/source/androidx.work/work-runtime/Additions/WorkAdditions.cs(152,25): error CS0117: 'WorkRequest.Builder' does not contain a definition for 'SetPeriodStartTime' [/Users/Shared/Projects/d/X/AX/wsu-20230222/generated/androidx.work.work-runtime/androidx.work.work-runtime.csproj]
/Users/Shared/Projects/d/X/AX/wsu-20230222/source/androidx.work/work-runtime/Additions/WorkAdditions.cs(155,25): error CS0117: 'WorkRequest.Builder' does not contain a definition for 'SetPeriodStartTime' [/Users/Shared/Projects/d/X/AX/wsu-20230222/generated/androidx.work.work-runtime/androidx.work.work-runtime.csproj]
/Users/Shared/Projects/d/X/AX/wsu-20230222/source/androidx.work/work-runtime/Additions/WorkAdditions.cs(158,25): error CS0117: 'WorkRequest.Builder' does not contain a definition for 'SetPeriodStartTime' [/Users/Shared/Projects/d/X/AX/wsu-20230222/generated/androidx.work.work-runtime/androidx.work.work-runtime.csproj]
    107 Warning(s)
    8 Error(s)
```

Reply:
Looks like these method(s) were originally `RestrictTo` and Google removed them, so there's nothing for us to do here:

![image](https://user-images.githubusercontent.com/179295/227992155-d27241f5-1a9c-46e3-8617-f05a46ca22bf.png)